crates/meilisearch/tests/tasks/mod.rs

```diff
@@ -50,8 +49,8 @@ async fn get_task_status() {
 
 #[actix_rt::test]
 async fn list_tasks() {
-    let server = Server::new().await;
-    let index = server.index("test");
+    let server = Server::new_shared();
+    let index = server.unique_index();
```

Comment:
```suggestion
    // Do not use a shared server because we want to assert stuff against the global list of tasks
    let server = Server::new().await;
    let index = server.index("test");
```